.travis.yml

```diff
@@ -24,7 +22,18 @@ matrix:
   - rvm: 2.4.0
     gemfile: gemfiles/rails_6.0.gemfile
 before_install:
-  - sudo service postgresql stop && sudo service postgresql start $POSTGRESQL_VERSION
+  - wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+  - echo "deb http://apt.postgresql.org/pub/repos/apt/ `lsb_release -cs`-pgdg main" | sudo tee /etc/apt/sources.list.d/pgdg.list
+  - sudo apt update
+  - sudo apt install -yq --no-install-suggests --no-install-recommends postgresql-${POSTGRESQL_VERSION} postgresql-client-${INSTALL_VERSION}
```

Comment:
@betesh 
`INSTALL_VERSION`
Is this typo?

Reply:
Yes.  Should be POSTGRESQL_VERSION

Reply:
Fixed in #33